applications/services/desktop/views/desktop_view_locked.c

```diff
@@ -121,7 +121,7 @@ static void desktop_view_locked_draw(Canvas* canvas, void* model) {
     if(view_state == DesktopViewLockedStateDoorsClosing) {
         desktop_view_locked_doors_draw(canvas, m);
         canvas_set_font(canvas, FontPrimary);
-        elements_multiline_text_framed(canvas, 42, 30 + STATUS_BAR_Y_SHIFT, "Locked");
+        elements_multiline_text_framed(canvas, 42, 24 + STATUS_BAR_Y_SHIFT, "Locked");
```

Comment:
You are right that it's slightly off, but changes you've made are not fixing issue.

There are some layouting and logical issues in elements_multiline_text_framed:
- x, y are actually not top left corner
- text is shifted to the top

Also popups were expected to be aligned in their own area(fullscreen minus statusbar)

Reply:
> You are right that it's slightly off, but changes you've made are not fixing issue.
> 
> There are some layouting and logical issues in elements_multiline_text_framed:
> 
>     * x, y are actually not top left corner
> 
>     * text is shifted to the top
> 
> 
> Also popups were expected to be aligned in their own area(fullscreen minus statusbar)

Based on my testing, it appears that x and y are located in the top left corner. I'm not entirely sure what you mean by "text is shifted to the top." Personally, I think that having these popups in the middle of the screen feels natural.